prow/config.yaml

```diff
@@ -14142,6 +14154,11 @@ periodics:
     preset-service-account: true
   spec:
     containers:
-    - image: gcr.io/minigo-testing/minigo-prow-harness:latest
+    - image: gcr.io/minigo-testing/minigo-prow-harness-v2:latest
       imagePullPolicy: Always
-
+      args:
+      - "--clean"
+      - "--job=$(JOB_NAME)"
+      - "--repo=tensorflow/minigo=$(PULL_REFS)"
```

Comment:
periodic job does not have $PULL_REFS, just use master

Reply:
How's this?